### ISSUE TITLE
Negative habit score should start from 0 instead of 100

### ISSUE THREAD

**Issue by krafi99** [iSoron/uhabits#1642]
## Pre-submission checklist
- [x] I am submitting a bug report, not a feature request.

- [x] I have searched for similar issues, but did not find any matches.

## Description
Score of negative habits, i.e. habits that have a maximum limit, should start from 0 instead of 100 as of now. Once I have successfully fulfilled my target for a certain period, as in normal habits, it should reach 100.

## Steps to reproduce
1. Create a new habit with target type 'at most'
2. Set a value
3. Tap on the habit
4. Score starts from 100

## System information
 - Phone: Xiaomi Redmi 8A
 - Phone Operating System: Android 10
 - App version: 2.1.1

## Screenshots
![IMG_20230209_153217](https://user-images.githubusercontent.com/63806657/217773470-157d1ad5-aa99-434f-9b7a-0519b29c810a.jpg)

**Comment by aurisnoctis** [comment]
Thank you for loop habit / uhabits!

For "at most" habits, the start at 100@% score is indeed confusing. At first, as I get better, my score deteriorates, starting from 100%. I understand you use the same logic and smoothing in all settings, which is a nice consistency. But in this case, to see getting better in the score over time when you are gradually improving would be more motivating! 🙌

@iSoron At some point you were considering to think about it if I understand this comment correctly https://github.com/iSoron/uhabits/pull/1106#issuecomment-1008724242

**Comment by aurisnoctis** [comment]
I am tracking a "maximum" goal. I input my values every day. I have not been below my set threshold a single time. But my score started at 100%. And is now at about 93%. That doesn't make sense actually.

**Comment by herrbrixter** [comment]
I agree with @aurisnoctis  . AT_MOST measureable habits should start from 0%.
